package.json

```diff
@@ -18,6 +18,7 @@
   "dependencies": {
     "@molgenis/molgenis-api-client": "^3.0.0",
     "@molgenis/molgenis-i18n-js": "^0.0.5",
+    "bootstrap-vue": "^2.0.0-rc.11",
```

Comment:
are we oke with 'rc' ??  we have run into issues before..

Reply:
AFAIK This is the most stable version that's compatible with bootstrap 4